Languages/JS/questions.json

```diff
@@ -81,6 +81,33 @@
 		"trueOrFalse": false,
 		"shortAns": "BOM stands for Browser Object Model. It provides interaction with the browser. The default object of a browser is a window.",
 		"tag": "JavaScript"
-		}
+	},
+	{
+		"id": 9,
+		"question": "What does being weakly typed mean for JavaScript?",
+		"details": "Strict typing is a method in programming languages that assigns each variable a data type that any assignment must adhere to. Weak typed languages, such as JavaScript, do not enforce any such rule. This means a variable can be assigned as an integer, and a string can be concatenated to it, casting the entire thing to a string.",
+		"questionType": "multi",
+		"trueOrFalse": false,
+		"shortAns": "Weakly typed means that you can assign a variable without delclaring what data type the variable uses.",
```

Comment:
please do not put the  wording of the question in the shortAns for the question, it makes it really obvious that this is the correct answer.

Reply:
Apologies, this should follow format better: [f62de69](https://github.com/Tc2r1/Hacktoberfest_Interview_App/pull/79/commits/f62de6901b54495875aba00059601d87fcb95ddf)